### PR TITLE
[KYUUBI #517][LICENSE] Add license header on kyuubi ASCII Logo

### PR DESCRIPTION
**Pull request by alex601049502**
### _Why are the changes needed?_

Add license header on kyuubi ASCII Logo.

### _How was this patch tested?_
Existing UT.

- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.readthedocs.io/en/latest/tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: bin/kyuubi [hunk 1]
```diff
@@ -209,3 +209,16 @@ case $1 in
...
+echo " /\ \/\ \                         /\ \      __ "
+echo " \ \ \/'/'  __  __  __  __  __  __\ \ \____/\_\ "
+echo "  \ \ , <  /\ \/\ \/\ \/\ \/\ \/\ \\\\\ \ '__\`\/\ \ "
+echo "   \ \ \\\\\\\`\\\\\ \ \_\ \ \ \_\ \ \ \_\ \\\\\ \ \L\ \ \ \ "
+echo "    \ \_\ \_\/\`____ \ \____/\ \____/ \ \_,__/\ \_\ "
+echo "     \/_/\/_/\`/___/> \/___/  \/___/   \/___/  \/_/ "
+echo "                /\___/ "
+echo "                \/__/ "
+echo " "
+}
```
Comment (turboFei) [inline bin/kyuubi:224]: add a new line at the end of file

**Review by pan3793 — APPROVED** [review]
LGTM

**Review by yaooqinn — APPROVED** [review]

**Review by turboFei — APPROVED** [review]

**Comment by turboFei** [comment]
thanks, merging to master

**Comment by turboFei** [comment]
thanks for your contribution @alex601049502

**Comment by turboFei** [comment]
Hi @alex601049502 

It seems that your commit email is not set in your github account setting.

Kindly reminder that, you can refer below doc.

https://docs.github.com/en/github/setting-up-and-managing-your-github-user-account/setting-your-commit-email-address

**Comment by alex601049502** [comment]
> Hi @alex601049502
> 
> It seems that your commit email is not set in your github account setting.
> 
> Kindly reminder that, you can refer below doc.
> 
> https://docs.github.com/en/github/setting-up-and-managing-your-github-user-account/setting-your-commit-email-address

Thanks for reminding

**Pull request closed without merging**